Lib9c/Action/RankingBattle.cs

```diff
@@ -18,11 +18,13 @@
 namespace Nekoyume.Action
 {
     [Serializable]
-    [ActionType("ranking_battle10")]
+    [ActionType("ranking_battle11")]
     public class RankingBattle : GameAction
     {
         public const int StageId = 999999;
         public static readonly BigInteger EntranceFee = 100;
+        public const long UpdateTargetBlockIndex = 3_752_000L;
```

Comment:
I think `Update` is too general term. How about using the name suggesting that each arena is segmented? (e.g. `SegmentedArenaTargetBlockIndex`)

Or, In cases where proposals are separated by a name like EIP, [it is sometimes used as it is in the variable name](https://github.com/ethereum/go-ethereum/blob/8bbf83e7a4e680c3b3b8e7a1cb09b76479dd26fe/core/tx_pool.go#L591-L592).

Reply:
Is it better to use the lib9c issue number(e.g. `issue-883`)?

Reply:
GH issue number can be better but I think it isn't enough for naming... 

If we don't have a suitable name, it might be ok to keep the issue but leave a comment.

Reply:
ok, i'll leave a comment this.